component/scripts/approve.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+LAST_UPGRADE_JOB_TIMESTAMP=$(kubectl -n "${UPGRADE_CONTROLLER_NAMESPACE}" get upgradejobs.managedupgrade.appuio.io -ojson | jq -r '.items | sort_by(.metadata.creationTimestamp) | reverse | .[0] | .metadata.creationTimestamp')
+AIRLOCK_INSTALLPLAN=$(kubectl -n "${AIRLOCK_NAMESPACE}" get ip -ojson | jq -r --arg upgrade_job "${LAST_UPGRADE_JOB_TIMESTAMP}" '.items | sort_by(.metadata.creationTimestamp) | reverse | [.[] |  select((.spec.approved != true) and (.metadata.creationTimestamp < $upgrade_job))][0] | .metadata.name')
```

Comment:
All hook jobs get all details about their associated upgradejob as environment variables which are injected by the upgrade controller, e.g. creation timestamp of the upgrade job is available as envvar `${JOB_metadata_creationTimestamp}`.

```suggestion
AIRLOCK_INSTALLPLAN=$(kubectl -n "${AIRLOCK_NAMESPACE}" get ip -ojson | jq -r --arg upgrade_job "${JOB_metadata_creationTimestamp}" '.items | sort_by(.metadata.creationTimestamp) | reverse | [.[] |  select((.spec.approved != true) and (.metadata.creationTimestamp < $upgrade_job))][0] | .metadata.name')
```